Tests/scripts/runTests.py

```diff
@@ -103,6 +106,7 @@ def RunTestCase(asn1, acn, behavior, expErrMsg):
             PrintFailed("coverage failed. (less than 100%)")
             sys.exit(1)
     else:
+        prevDir = os.getcwd()
         os.chdir(targetDir)
```

Comment:
On mac os, this changed from **Tests/tmp** to **Tests/scripts**, and line 160 went up instead of back, which breaks the next test since it won't find **Asn1f2.exe**.